frontend/ui/lib/widgets/place_block_widget.dart

```diff
@@ -13,17 +13,40 @@ class _PlaceBlockWidgetState extends State<PlaceBlockWidget> {
   final String placeName;
   _PlaceBlockWidgetState(this.placeName);
 
+  List<bool> _selections = List.generate(3, (_) => false);
```

Comment:
We'll need to take care that this stays in sync with the number of toggle buttons below. Doing so might be easier if the buttons were extracted into their own widget (not sure that's worth doing here).

Reply:
Shouldn't the amount of toggle buttons stay the same?

Reply:
It might, but it might also evolve over time. For example, if you do move the delete button outside of the toggle button group and into its own button (not sure if you plan to do that in this PR or in a follow-up), you'll need to update the list size. This list and the toggle buttons are effectively coupled, so it's best to keep them as close together as possible to ensure they stay in sync. Ideally, the code would be structured in such a way that desyncing is _impossible_ (though I'm not sure if there's a nice way to do that in this situation).